lib/src/core/channel/guild/forum/forum_channel.dart

```diff
@@ -1,18 +1,49 @@
+import 'package:nyxx/src/core/channel/guild/forum/forum_channel_tags.dart';
 import 'package:nyxx/src/core/channel/guild/forum/forum_tag.dart';
 import 'package:nyxx/src/core/channel/guild/guild_channel.dart';
 import 'package:nyxx/src/core/channel/thread_channel.dart';
 import 'package:nyxx/src/core/channel/thread_preview_channel.dart';
+import 'package:nyxx/src/core/message/emoji.dart';
+import 'package:nyxx/src/core/message/guild_emoji.dart';
+import 'package:nyxx/src/core/message/unicode_emoji.dart';
 import 'package:nyxx/src/core/snowflake.dart';
+import 'package:nyxx/src/internal/exceptions/unknown_enum_value.dart';
 import 'package:nyxx/src/internal/interfaces/mentionable.dart';
 import 'package:nyxx/src/internal/response_wrapper/thread_list_result_wrapper.dart';
 import 'package:nyxx/src/nyxx.dart';
 import 'package:nyxx/src/typedefs.dart';
 import 'package:nyxx/src/utils/builders/forum_thread_builder.dart';
 
+enum ForumSortOrder {
+  /// Sort forum posts by activity
+  latestActivity(0),
+
+  /// Sort forum posts by creation time (from most recent to oldest)
+  creationDate(1);
+
+  final int value;
+  const ForumSortOrder(this.value);
+
+  static ForumSortOrder _fromValue(int value) => values.firstWhere((v) => v.value == value, orElse: () => throw UnknownEnumValueError(value));
+
+  @override
+  String toString() => 'ForumSortOrder[$value]';
+}
```

Comment:
```suggestion
enum ForumSortOrder {
  /// Sort forum posts by activity
  latestActivity,

  /// Sort forum posts by creation time (from most recent to oldest)
  creationDate;

  static ForumSortOrder _fromValue(int value) => values.firstWhere((v) => v.index == value, orElse: () => throw UnknownEnumValueError(value));

  @override
  String toString() => 'ForumSortOrder[$index]';
}
```

Reply:
I know it auto assigns values but its going to be explicit with values if anything changes in dart later on, or if someone inserts value in wrong spot

Reply:
All right then 👍